src/components/editor/EditAuthorsForm.jsx

```diff
@@ -84,8 +41,8 @@ export default class EditAuthorsForm extends BaseComponent {
               <div key={author.slug}>
                 {
                   !striked
-                    ? <a style={{float: "left", cursor: "pointer"}} aria-label="Remove Author Button" onClick={this.handleDeleteAuthor.bind(this, author.slug, true)}>&times;&nbsp;</a>
-                    : <a style={{float: "left", cursor: "pointer"}} aria-label="Remove Author Button" onClick={this.handleUnstrikeAuthor.bind(this, author.slug)}>~&nbsp;</a>
+                    ? <a style={{float: "left", cursor: "pointer"}} aria-label="Remove Author Button" onClick={this.props.handleDeleteAuthor.bind(null, author.slug, true)}>&times;&nbsp;</a>
```

Comment:
Hmm, I don't know if there's other effects of not including an href, but if you're not gonna include the href there's no point having an a tag at all. You could just use span for example. I think you can return false on the onClick handler normally to override actually going to the link, but I'm not sure if react plays with that a bit more. 

It's not that big of a deal, but try to rely on native parts as much as possible. You never know what some random android / iOS / Chroome / Safari does to "a" tags to make them "feel" natural. It might be undefined.